app/assets/stylesheets/modules/_selects.styl

```diff
@@ -35,4 +33,4 @@
   select.sorts
     background url('../images/sort.svg') no-repeat 48% white
     -webkit-appearance none
-    text-indent -99px
+    text-indent -200px
```

Comment:
@ragesoss Tested this CSS on Mac in Chrome, Firefox and Safari and it looks good. Not sure about other platforms